### PR TITLE
Refactor to swagger TS API

### PR DESCRIPTION
**Pull request by reivaj05**

### CODE REVIEW

**Review by bexsoft — DISMISSED** [review]
Please review ` Tags can be created and deleted` test, seems like this is a real failure

File: portal-ui/src/screens/Console/Common/FormComponents/RadioGroupSelector/RadioGroupSelector.tsx
Comment (bexsoft) [inline portal-ui/src/screens/Console/Common/FormComponents/RadioGroupSelector/RadioGroupSelector.tsx:1]: This file will be replaced by mds component & deleted soon

File: portal-ui/src/screens/Console/Configurations/TiersConfiguration/AddTierConfiguration.tsx [hunk 1]
```diff
@@ -181,19 +181,25 @@ const AddTierConfiguration = ({ classes }: IAddNotificationEndpointProps) => {
...
+        .addTier(payload)
         .then(() => {
           setSaving(false);
           navigate(IAM_PAGES.TIERS);
         })
-        .catch((err: ErrorResponseHandler) => {
+        .catch((err) => {
           setSaving(false);
-          dispatch(setErrorSnackMessage(err));
+          dispatch(setErrorSnackMessage(errorToHandler(err.error)));
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Configurations/TiersConfiguration/AddTierConfiguration.tsx:202]: This is not triggering the snack error message
Screenshot 2023-06-28 at 15 02 05

**Review by bexsoft — DISMISSED** [review]
A couple of issues that need to verify if are related to this change:

Enabling Bucket encryption with wrong configuration doesn't throw an error

Screenshot 2023-06-28 at 14 56 31

Restart hangs and doesn't remove the top notification when completed:
Screenshot 2023-06-28 at 15 07 16

**Comment by reivaj05** [comment]
@bexsoft issues fixed, the only one i wasn't able to reproduce was the issue with restarting, working fine for me

**Review by bexsoft — APPROVED** [review]

**Comment by cesnietor** [comment]
I just saw that the snack bar message doesn't show up when adding SSE-S3 encryption. It used to be shown.
before :
Screenshot 2023-07-05 at 1 44 44 PM
now:
Screenshot 2023-07-05 at 1 46 16 PM

**Comment by cesnietor** [comment]
I just debugged it a bit and is not being shown because error is null:
Screenshot 2023-07-05 at 3 34 43 PM

This is because of the following bug https://github.com/acacode/swagger-typescript-api/issues/536 cc @bexsoft @reivaj05

**Review by cesnietor — APPROVED** [review]
LGTM

**Review by bexsoft — APPROVED** [review]

**Pull request merged**